src/components/internshipCard/InternshipCard.jsx

```diff
@@ -43,13 +44,19 @@ export const InternshipCard = ({
         {getFieldLabel('internships.program.members')}
       </Typography>
     </CardContent>
-    <Box display="flex" flexDirection="row" justifyContent="center">
-      <Button className="internshipCardButton" size="small">
+    <Box textAlign="center" className="test">
+      <Link
+        to="/internshippage"
+        className="internshipCardButton"
+      >
         {getFieldLabel('internships.button.program.info')}
-      </Button>
-      <Button size="small">
+      </Link>
+      <Link
+        to={`/candidates/${id}`}
+        className="internshipCandidatesLink"
+      >
         {getFieldLabel('internships.button.candidates.list')}
```

Comment:
you can still put button on the inside, just don't provide any onClick

Reply:
ok